i18n/create_messages.py

```diff
@@ -87,7 +88,7 @@ def main():
   # Read in synonyms file, which must be output in every language.
   synonym_defs = read_json_file(os.path.join(
       os.curdir, args.source_synonym_file))
-  synonym_text = '\n'.join(['Blockly.Msg["{0}"] = Blockly.Msg["{1}"];'.format(
+  synonym_text = '\n'.join([u'/** @export */\nBlockly.Msg.{0} = Blockly.Msg.{1};'.format(
```

Comment:
Does @ export have to be on its own line?  That makes the resulting JS file quite bloated.  Can it be inline?